main.go

```diff
@@ -43,6 +44,8 @@ const (
 	DefaultDebugEnvironmentVariable = "DEBUG"
 	// DefaultBranchVersionEnvironmentVariable is the default environment variable that indicates the branch version to compare against
 	DefaultBranchVersionEnvironmentVariable = "BRANCH_VERSION"
+	// DefaultForkEnvironmentVariable is the default environment variable that indicates the fork URL
```

Comment:
None of these need to be exported since they can't be imported from main.

Reply:
Fixing